cocos/renderer/backend/metal/TextureMTL.mm

```diff
@@ -148,7 +168,7 @@ bool convertData(uint8_t* src, unsigned int length, TextureFormat format, uint8_
            [MTLTextureDescriptor texture2DDescriptorWithPixelFormat:Utils::toMTLPixelFormat(descriptor.textureFormat)
                                                               width:descriptor.width
                                                              height:descriptor.height
-                                                          mipmapped:TRUE];
+                                                          mipmapped:YES];
```

Comment:
If passing `YES` to mipmapped, then should we do mipmap manually too?

Reply:
The `mipmapped` parameter is there to tell Metal to calculate the number of mipmap levels that resulting image will have. And `generateMipmapsForTexture` method is used to generate mipmaps for a texture that already has mipmap levels.